install_config/router/f5_router.adoc

```diff
@@ -211,24 +209,24 @@ endif::[]
 
 As with the HAProxy router, the `oc adm router` command creates the service and
 deployment configuration objects, and thus the replication controllers and
-pod(s) in which the F5 router itself runs. The replication controller restarts
-the F5 router in case of crashes. Because the F5 router is watching routes,
-endpoints, and nodes and configuring *F5 BIG-IP®* accordingly, running the F5
-router in this way, along with an appropriately configured *F5 BIG-IP®*
+pod(s) in which the F5 router plug-in itself runs. The replication controller restarts
+the F5 router plug-in in case of crashes. Because the F5 router plug-in is watching routes,
+endpoints, and nodes and configuring *F5 BIG-IP* accordingly, running the F5
+router in this way, along with an appropriately configured *F5 BIG-IP*
 deployment, should satisfy high-availability requirements.
```

Comment:
s/should satisfy/"satisfies" or "satisfies most"

Reply:
agree